src/PerfView/EtwEventSource.cs

```diff
@@ -210,6 +210,22 @@ public override void ForEach(Func<EventRecord, bool> callback)
                             opcode = TraceEventOpcode.Stop;
                     }
 
+                    if (data.ProviderGuid == systemDataProviderGuid)
+                    {
+                        corelationOptions = CorelationOptions.UseActivityID;
+                        if ((int)data.ID == 1)          // BeginExecute
```

Comment:
💭 May be good to create internal static classes for constants like this. Future cleanup item since so many existing places already have the same code pattern.